test/run_test.js

```diff
@@ -1,22 +1,24 @@
 import { runTasks } from '../lib/run'
-// import { expect } from 'chai'
-// import { stub } from 'sinon'
-
-xdescribe('lib/run', () => {
+import childProcess from 'child_process'
+import { expect } from 'chai'
+import { spy } from 'sinon'
 
+describe('lib/run', () => {
   describe('runTasks', () => {
-    const tasks = [
-      'this:thing',
-      'that:thing'
-    ]
+    const tasks = ['dummy:1', 'dummy:2']
 
     before(() => {
+      spy(childProcess, 'exec')
```

Comment:
How do you feel about spying, but actually opening the process? I'm kind of torn. I like not having to overly mock it, but I don't like that the unit test depends on tasks in package.json.